### DISCUSSION TITLE
[Linux] Any input causes a tiny stutter (including outside of gameplay)

### DISCUSSION THREAD

**Discussion by OpenSauce04** [ppy/osu#22854] [Q&A] [answered]
### Type

Performance

### Bug description

Pressing any button, be that on a keyboard or mouse, causes a tiny stutter that lasts for about a frame, regardless of if the button you pressed does anything. This issue is basically invisible (to me at least) in osu!standard, however in other modes, it is much more visible due to their scrolling nature. I can confirm that this happens in all modes by using the FPS counter as reference

This issue also occurs if you press keys while a replay is playing, however no stutters occur due to the actions of replay itself, so this could possibly be a game-wide issue, rather than just existing within gameplay.

I can replicate this issue on both the official AppImage as well as the unofficial Flatpak release

My Linux distro is Archcraft

### Version

2023.305.0

### Logs

[runtime.log](https://github.com/ppy/osu/files/10951589/runtime.log)

**Comment by OpenSauce04** [comment]
Using the FPS counter as reference, this also appears to happen on the title screen

**Comment by peppy** [comment]
I’d recommmend providing a video, but this sounds like something you are going to ultimately have to figure yourself (it will be a configuration issue with your linux system)

**Comment by GoldM-amberdrake** [comment]
Are you using a realtime kernel? If so, it might cause stutters and performance issues.

**Reply by OpenSauce04** [reply]
No, I was just using the regular 6.2 kernel

**Comment by OpenSauce04** [comment, marked as answer]
I can confirm that this issue does not exist on Ubuntu 22.04. Must be something to do with my install

**Reply by OpenSauce04** [reply]
Also does not appear to occur on a fresh Arco install w/ KDE Plasma
Perhaps this was an issue specific to BSPWM
